### ISSUE TITLE
`@molt/core` and `@molt/cli` on jsr.io are NOT working

### ISSUE THREAD

**Issue by hasundue** [hasundue/molt#160]
Currently, `@molt/core` and `@molt/cli` at version `0.18.0` on [jsr.io](jsr.io) are broken. Running `@molt/cli` results in the following error, for example:

```
> molt deno.json
Failed fetching. Retrying in 250ms...
Failed fetching. Retrying in 500ms...
Failed fetching. Retrying in 1000ms...
```

I'm working on fixing this right now and will release `0.18.1` soon, hopefully.

**Comment by hasundue** [comment]
Can't get `jsr:@deno/graph` work. I created an issue on the original repo: https://github.com/denoland/deno_graph/issues/453

**Comment by hasundue** [comment]
It was mostly my mistake and now the migration to `jsr:@deno/graph` is done. Will release `0.18.1` soon.

**Comment by hasundue** [comment]
Released 0.18.0.
